tests/Cookie/Middleware/EncryptCookiesTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Illuminate\Tests\Cookie\Middleware;
 
+use Illuminate\Encryption\Key;
```

Comment:
I think the way imports are organized in Laravel is by size, not alphabetically. May be wrong.